py/torch_tensorrt/dynamo/conversion/ops_evaluators.py

```diff
@@ -47,3 +47,81 @@ def aten_ops_arange_start_step(
     name: str,
 ) -> Union[TRTTensor, Sequence[TRTTensor]]:
     return np.arange(*args)
+
+
+def rand_validator(rand_node: Node) -> bool:
+    dtype = rand_node.kwargs.get("dtype", None)
+    layout = rand_node.kwargs.get("layout", None)
+    if dtype is not None:
+        _LOGGER.debug(
+            f"Currently we don't support specifying output dtype, got {dtype}."
+        )
+        return False
+    if layout is not None:
+        _LOGGER.debug(f"Currently we don't support specifying layout, got {layout}.")
+        return False
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.rand.default)
+def aten_ops_rand(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    device = kwargs.get("device", None)
+    return np.random.rand(*args).to(device=device)
```

Comment:
As above, I think you would need to unpack the integers with something like `np.random.rand(*args[0])`. Additionally, `.to` would not have effect in Numpy

Reply:
Thanks for the review. I think `np.random.rand(*args) ` should also work, since it passes locally. Let me cross check.